Added extra symbols for LaTeX editor

junthehacker opened the pull request:
* Added more symbols: `=+-*/^_{}`.
* Shinked padding to optimize for small screens.
* Fixed `ui.js` bugs.
![image](https://user-images.githubusercontent.com/7855724/36072635-d05afc3a-0ef0-11e8-9b52-2d245b5b979b.png)

hughdani commented on public/app/js/ui.js:
```diff
@@ -18,28 +18,45 @@ onMainLoop(function(){
+            // Include all tool buttons
+            $.each($(".message-tool-button").not(".latex-editor-symbol-button"), function(key, val){
+                toolButtonTotalWidth += $(val).outerWidth();
+                console.log(val);
```
Don't forget to remove this.

![image](https://user-images.githubusercontent.com/31714259/36079643-68f76b9c-0f53-11e8-8a38-9dafb58d0cf6.png)

junthehacker commented:
* I have fixed the iPhone 5 sizing issue. the problem actually only happens if you use the Chrome simulator. If you use an actual iPhone 5 / 5c / 5s / SE, or iOS virtual machine, the UI displays correctly. but fixed anyways.
* Removed the log.
